tests/OpenCloud/Tests/Database/Resource/BackupTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright 2012-2014 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCloud\Tests\Database\Resource;
+
+use OpenCloud\Tests\Database\DatabaseTestCase;
+
+class BackupTest extends DatabaseTestCase
+{
+    public function test_Class()
+    {
+        $this->assertInstanceOf('OpenCloud\Database\Resource\Backup', $this->backup);
+    }
+
+    public function testDelete()
```

Comment:
maybe also test backup creation? specifically hitting those edge cases that will throw errors